src/main.cpp

```diff
@@ -28,26 +28,40 @@ CANSignal<uint8_t, 0, 8, CANTemplateConvertFloat(1), CANTemplateConvertFloat(0),
 // TX CAN Message (Might need to edit transmit period (currently 100))
 CANTXMessage<1> tx_message{can_bus, 0x300, 1, 100, read_timer, cur_throttle_signal};
 
+// TX CAN Signal for motor temp
 CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> motor_temp_signal{};
-CANSignal<float, 16, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> coolant_temp_signal{};
-CANSignal<float, 32, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> ambient_temp_signal{};
-CANRXMessage<3> rx_message{can_bus, 0x420, motor_temp_signal, coolant_temp_signal, ambient_temp_signal};
+// CANSignal<float, 16, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> coolant_temp_signal{};
+// CANSignal<float, 32, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> ambient_temp_signal{};
+
+// TX CAN Signal for battery amperage and voltage
+CANSignal<float, 48, 16, CANTemplateConvertFloat(0.01), CANTemplateConvertFloat(0), false> battery_amperage_signal{};
+CANSignal<float, 24, 16, CANTemplateConvertFloat(0.01), CANTemplateConvertFloat(0), false> battery_voltage_signal{};
+
+// CANRXMessage
+CANRXMessage<1> motor_message{can_bus, 0x420, motor_temp_signal};
+CANRXMessage<2> amp_message{can_bus, 0x240, battery_amperage_signal, battery_voltage_signal};
 
 void ReadAcceleratorPress() {
-	cur_throttle_signal = throttle.ReadAcceleratorPress();
+	cur_throttle_signal = throttle.ReadAcceleratorPress(motor_temp_signal, battery_amperage_signal, battery_voltage_signal);
 	Serial.print("cur_throttle_signal: ");
```

Comment:
could you define a "debug_mode" variable and only serial print when you are in debug mode